src/frontend/src/components/Posts.svelte

```diff
@@ -52,11 +54,12 @@
       </div>
     {/if}
     <!-- Username & Bio Container -->
-    <div class="ml-4">
+    <div class="ml-4 overflow-hidden first-letter:">
         <h1 class="text-xl font-bold">@{ profile.username }</h1>
-        <p class="text-sm">{ profile.about }</p>
+        <span class="npub" title="Your public key.">{npub}</span>
```

Comment:
nezobrazí mi ten reťazec celý, odreže mi ho

Reply:
To ellipsis neudělalo to bych chtěl, ideální by bylo zalomit na nový řádek, ale dělá to to, že to přesahuje ten parent element :/ nicméně když na to klikneš a kopíruješ to tak to úlohu splní i s overflow-hidden.